ipaserver/install/dsinstance.py

```diff
@@ -753,8 +753,14 @@ def __config_dns_module(self):
     def __config_lockout_module(self):
         self._ldap_mod("lockout-conf.ldif")
 
-    def __config_graceperiod_module(self):
-        self._ldap_mod("graceperiod-conf.ldif")
+    def _config_graceperiod_module(self):
```

Comment:
Nitpick but since the method is called from upgrade.py, I would name it `config_graceperiod_module` (without the leading underscore).